### DISCUSSION TITLE
Deploy Piranha CMS with MySQL on Ubuntu for production

### DISCUSSION THREAD

**Discussion by Spec86** [PiranhaCMS/piranha.core#1890] [Q&A]
Hello, how are deploy Piranha CMS with MySQL on Ubuntu? Not local deploy. Deploy on production server. Step by step. May be you have Ansible file with code comments for this procedure?

**Comment by tidyui** [comment]
@jensbrak Do you have any tips for this, if I remember correctly you’re running your site on this setup!

**Reply by Spec86** [reply]
I need basic Ansible file for deploy Piranha CMS on Ubuntu.

**Reply by jensbrak** [reply]
Not using Red Hat or Ansible but I do run Piranha on Debian (Raspberry Pi OS to be precise) using MySQL, Nginx and more. Happy to share anything regarding this if it may help.

**Reply by Spec86** [reply]
Can you publish your Ansible file in GitHub or Ansible Galaxy? Phone. Dublicate write.

**Reply by tidyui** [reply]
@Spec86 If I’m not misreading, @jensbrak clearly wrote that he’s *not* using Ansible.

**Reply by Spec86** [reply]
Sorry, I'not native English.

**Reply by Spec86** [reply]
I write in phone. Difficult translate in Google browser from GitHub app in iOS.

**Comment by Spec86** [comment]
Can you publish your Ansible file in GitHub or Ansible Galaxy?
